pythonforandroid/recipes/python3/patches/reproducible-buildinfo.diff

```diff
@@ -1,13 +1,11 @@
-# DP: Build getbuildinfo.o with DATE/TIME values when defined
-
---- a/Makefile.pre.in
-+++ b/Makefile.pre.in
-@@ -785,6 +785,8 @@ Modules/getbuildinfo.o: $(PARSER_OBJS) \
+--- Python-3.11.4/Makefile.pre.in	2023-06-07 03:30:27.000000000 +0530
++++ Python-3.11.4.mod/Makefile.pre.in	2023-07-08 11:38:12.788515487 +0530
+@@ -1238,6 +1238,8 @@
  	      -DGITVERSION="\"`LC_ALL=C $(GITVERSION)`\"" \
  	      -DGITTAG="\"`LC_ALL=C $(GITTAG)`\"" \
  	      -DGITBRANCH="\"`LC_ALL=C $(GITBRANCH)`\"" \
-+	      $(if $(BUILD_DATE),-DDATE='"$(BUILD_DATE)"') \
-+	      $(if $(BUILD_TIME),-DTIME='"$(BUILD_TIME)"') \
++        $(if $(BUILD_DATE),-DDATE='"$(BUILD_DATE)"') \
++        $(if $(BUILD_TIME),-DTIME='"$(BUILD_TIME)"') \
  	      -o $@ $(srcdir)/Modules/getbuildinfo.c
  
- Modules/getpath.o: $(srcdir)/Modules/getpath.c Makefile
+ Modules/getpath.o: $(srcdir)/Modules/getpath.c Python/frozen_modules/getpath.h Makefile $(PYTHON_HEADERS)
```

Comment:
Did not the original patch work here?

Reply:
No was just afraid of upcoming errors, :) reverting.